nextcloudappstore/core/models.py

```diff
@@ -56,18 +56,12 @@ def can_delete(self, user: User) -> bool:
 
     @staticmethod
     def search(lang, terms):
```

Comment:
Right, the question really is how that would work out with django-parler, since that one also overrides the default manager afaik and what the benefits would be since its basically only a single query.